### DISCUSSION TITLE
Cant access 6.2 Geoservers

### DISCUSSION THREAD

**Discussion by khartman-nj** [ua-gist604b-f24/syllabus#34] [General]
I was trying to run thought 6.2 but cant access the geoserver all I get is Page isnt working all the code worked right I just cant get into the servers. Any suggestions on how to fix this ?

**Comment by travisz09** [comment]
Mine was working earlier tonight, but then as I was trying to troubleshoot a different issue I lost access to my geoserver page as well. All I could see was a white screen with a http 505 error. 

I'm not sure, but I wonder if the server is overwhelmed? I'm going to try again in the morning.

**Reply by travisz09** [reply]
I was able to complete the assignment a few moments ago. I didn't do anything differently, but it seemed to be working better this morning. IDK what the difference was, other than the timing, I guess.
